lib/sdl2/image.nit

```diff
@@ -75,4 +75,8 @@ extern class SDLImgInitFlags `{ int `}
 
 	# Add the WEBP support to this flag set
 	fun webp: SDLImgInitFlags `{ return self | IMG_INIT_WEBP; `}
+
+	redef fun hash `{ return self; `}
+	redef fun ==(o) do return o isa SDLImgInitFlags and native_equals(o)
+	private fun native_equals(o: SDLImgInitFlags): Bool `{ return self == o; `}
```

Comment:
If these are standard ffi definition, are they documented? Why are not in Pointer?